stripe/src/main/java/com/stripe/android/net/StripeApiHandler.java

```diff
@@ -151,11 +177,11 @@ public static Token retrieveToken(
         return headers;
     }
 
-    private static String getApiUrl() {
+    static String getApiUrl() {
```

Comment:
Only getApiUrl, actually, since I'm testing that one -- should have added the flag for @VisibleForTesting